benchmarks/Sentry.Benchmarks/RetryAfterHandlerOverhead.cs

```diff
@@ -10,10 +10,10 @@ namespace Sentry.Benchmarks
     public class RetryAfterHandlerOverhead
     {
         private HttpMessageInvoker _invoker;
-        private readonly HttpRequestMessage _request = new HttpRequestMessage(HttpMethod.Get, "/");
+        private readonly HttpRequestMessage _request = new(HttpMethod.Get, "/");
 
         [Params(1, 10, 100)]
-        public int RequestCount;
+        public int RequestCount { get; set; }
```

Comment:
I think this was on purpose so it's a field access vs a method one

Reply:
Oh